MIP101/MIP101.md

```diff
@@ -706,10 +487,6 @@ FacilitatorDAOs must err on the side of caution and act in case there is any kin
 
 All FacilitatorDAOs must immediately either support the action or propose overturning it and penalizing the FacilitatorDAO when an operational security breach derecognition is initiated. Failing to act is itself misalignment and results in penalties for all the FacilitatorDAOs that failed to act.
 
-#### 2.6.7: Aligned Delegate Charity
-
-While ADs are not allowed to be involved in operational activities, or allocate their AD income towards operational activities that goes beyond their task of providing research and information to AVCs, the one exception is using a portion of their AD income for donations to charity. It is encouraged that ADs compete and campaign on their ability to achieve some level of charitable impact with the resources available to them, in order to demonstrate alignment with the public good purpose of the DAO.
-
 ### 2.7: FacilitatorDAOs and Facilitators
 
 FacilitatorDAOs are a type of SubDAO that can be given responsibility over MakerDAO Scopes and SubDAO Scopes in return for tokenomics rewards.
```

Comment:
Comment for the point below (2.7.1): I believe FacilitatorDAOs and Facilitators are described in Article 7 of the Governance Scope. Article 6 refers to ADs.